### PR TITLE
🐛 Fix/develop

### PR DESCRIPTION
**Pull request by Lyokolux**
## Checklist

- [x] I compare my branch with `develop`
- [x] All tests passed after running `yarn s-check && yarn lint`

### CODE REVIEW

**Comment by Lyokolux** [comment]
@johannchopin Clean !

File: src/components/QuestionsForm/Question.svelte [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <script lang="ts">
   import { reverseArray } from '../../utils'
-  import type { Question } from './QuestionsForm.svelte'
+  import type { CustomQuestion } from '../../pages/UserInfos/questions/ExperienceGrade.svelte'
 
-  export let question: Question
+  export let question: CustomQuestion
```
Comment (johannchopin) [inline src/components/QuestionsForm/Question.svelte:5]: Please update the type `Question` to have `ids` **AND** `labels`

File: src/const.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ export const KEY = {
   tab: 'Tab'
 }
 
-export const QUALIFICATION = ['pleasant', 'unpleasant', 'simple', 'complicated', 'practical', 'not-practical', 'tedious', 'effective', 'good', 'bad', 'motivating', 'discouraging'] as const
+export const QUALIFICATION = ['pleasant', 'unpleasant', 'simple', 'complicated', 'practical', 'not-practical', 'tedious', 'effective', 'good', 'bad', 'motivating', 'discouraging', 'any_experience', 'lot_of_experience'] as const
```
Comment (johannchopin) [inline src/const.ts:11]: Use the `not-practical` or the `any_experience` syntax?

File: src/store/api.ts [hunk 3]
```diff
@@ -24,7 +24,7 @@ function createApi() {
    * @param currentUserDoc provided in the store
    */
   // TODO: type experiment when the type will be defined
-  const addExperimentRequest = async (userDoc: DocumentReference, experiment) => (
+  const addExperimentRequest = async (userDoc: DocumentReference, experiment: any) => (
```
Comment (johannchopin) [inline src/store/api.ts:27]: Please add the type

**Review by johannchopin — CHANGES_REQUESTED** [review]

**Comment by johannchopin** [comment]
🙏

**Pull request merged**